RSA/simple2.c

```diff
@@ -0,0 +1,192 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <math.h>
+
+//функция, котоая проверяет число на простоту
+int isPrime(long int n){
+        for (long int i = 2; i*i <= n; ++i){
+                if (n % i == 0) {
+                        return 0;
+                }
+        }
+        return 1;
+}
+
+//функция, которая возвращает простое рандомное число длины,
+//введенной с клавиатуры и больше
+//upd: длина простого числа определяется рандомно
+long int lookingForSimple() {
+
+        int n;
```

Comment:
Why not just `int n = rand() % 7`. Also I have to mention that `%7` is not scalable.